dace/transformation/interstate/sdfg_nesting.py

```diff
@@ -117,24 +94,25 @@ def apply(self, sdfg):
         for oldarrname, newarrname in transients.items():
             nested_sdfg.arrays.pop(oldarrname)
             nested_sdfg.arrays[newarrname].transient = False
-            outer_sdfg.arrays[oldarrname].transient = False
+            outer_sdfg.arrays[oldarrname].transient = True
```

Comment:
What's this doing? Do we even need this line, e.g. in the case that the outer array is not transient?

Reply:
Probably it is not needed. Putting it to `False` was wrong by the way, as additional kernel parameters were created

Reply:
ok. Can you try removing this line and see if the tests still pass?

Reply:
I confirm that that was unecessary